From: Dominique Leuenberger <dimstar@opensuse.org>
Date: Thu, 17 May 2018 17:40:37 +0200
Subject: qt: fix build against Qt 5.11

Qt 5.11 is in RC phase now, and some headers need to be explicitly
included in order to build successfully.

Signed-off-by: Jean-Baptiste Kempf <jb@videolan.org>
---
 modules/gui/qt/components/simple_preferences.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules/gui/qt/components/simple_preferences.cpp b/modules/gui/qt/components/simple_preferences.cpp
index a89935d..43d9410 100644
--- a/modules/gui/qt/components/simple_preferences.cpp
+++ b/modules/gui/qt/components/simple_preferences.cpp
@@ -36,6 +36,7 @@
 #include <QString>
 #include <QFont>
 #include <QToolButton>
+#include <QButtonGroup>
 #include <QSignalMapper>
 #include <QVBoxLayout>
 #include <QScrollArea>
